doc/classes/DirAccess.xml

```diff
@@ -14,7 +14,7 @@
 		# Static
 		DirAccess.make_dir_absolute("user://levels/world1")
 		[/codeblock]
-		[b]Note:[/b] Many resources types are imported (e.g. textures or sound files), and their source asset will not be included in the exported game, as only the imported version is used. Use [ResourceLoader] to access imported resources.
+		[b]Note:[/b] Accessing project ("res://") directories once exported may exhibit unexpected behavior as some files are converted to engine-specific formats and their original source files may not be present in the expected PCK package. To access project resources once exported, it is recommended to use [ResourceLoader] instead of [FileAccess] because of this reason.
```

Comment:
```suggestion
		[b]Note:[/b] Accessing project ("res://") directories once exported may behave unexpectedly as some files are converted to engine-specific formats and their original source files may not be present in the expected PCK package. Because of this, to access resources in an exported project, it is recommended to use [ResourceLoader] instead of [FileAccess].
```